src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -83,24 +81,7 @@ public void c024928()
         [Fact]
         public void TestBinary()
```

Comment:
Note, there is another PR that disables the test make sure the test is enabled once this change is merged.